stable/spinnaker/values.yaml

```diff
@@ -80,9 +77,20 @@ minio:
   serviceType: ClusterIP
   accessKey: spinnakeradmin
   secretKey: spinnakeradmin
+  bucket: "spinnaker"
   nodeSelector: {}
 
 gcs:
   enabled: false
   project: my-project-name
+  bucket: "<GCS-BUCKET-NAME>"
   jsonKey: '<INSERT CLOUD STORAGE JSON HERE>'
+
+s3:
+  enabled: false
+  bucket: "<S3-BUCKET-NAME>"
```

Comment:
I'm not the worlds biggest fan of `<THESE-DEFAULT-PLACEHOLDERS>` but I don't have a good alternative. I figure that this can be improved in a later pass if it ever is an issue.

Reply:
Agreed.